File: JagWebApp/ClientApp/src/app/test-stub/menu-item.stub.ts
import { MenuItem } from "../models/menuItem";

export const menuItemStub: MenuItem = {
    id: 1,
    dish: {
        name: 'a',
        category: {
            id: 1,
            name: 'b'
        },
        price: 1,
        amount: 1
    },
    limit: 1,
    available: 1
};

File: JagWebApp.Tests/Stubs/MenuItemStub.cs
using JagWebApp.Core.Models;
using System.Collections.ObjectModel;

namespace JagWebApp.Tests.Stubs
{
    class MenuItemStub
    {
        public static MenuItem GetMenuItem()
        {
            return new MenuItem
            {
                Dishes = new Collection<MenuItemDish>
                {
                    new MenuItemDish
                    {
                        DishId = 1,
                        Dish = DishStub.GetDish()
                    }
                },
                Price = 1,
                Available = 1,
                IsMain = true
            };
        }
    }
}
